src/sentry/api/event_search.py

```diff
@@ -229,6 +233,7 @@ class SearchVisitor(NodeVisitor):
     # A list of mappers that map source keys to a target name. Format is
     # <target_name>: [<list of source names>],
     key_mappings = {}
+    duration_keys = set(["transaction.duration", "p75", "p95", "p99"])
```

Comment:
How do you see this working with functions that sometimes use duration and sometimes don't? For example `avg(transaction.duration)`

Reply:
🤔 What are the cases where `avg(transaction.duration)` won't use duration?

Reply:
You could give it `stack.colno`

Reply:
Ah hm, someone could do `avg(stack.colno):>500s` and it would become `avg(stack.colno):>500000` which would be confusing (both in why it changes and why a user entered a column number greater than 500 seconds)
I'll look into adding something that will parse the column out of the function and check the `duration_keys`

Reply:
I added a `resolve_function` in event_search that does all the column parsing and column validation, so I would update that to validate the column passed in.

Reply:
`event_search` no longer has the pXX aliases anywhere, so these should be removed.